.github/workflows/main.yml

```diff
@@ -9,11 +9,10 @@ jobs:
     strategy:
       matrix:
         ruby:
-          - '3.0.7'
           - '3.1.6'
-          - '3.2.4'
-          - '3.3.3'
-
+          - '3.2.7'
```

Comment:
BTW I'm curious whether Github will use the most-recent patch versions if we would just specify `major.minor` ones (e.g. `3.1`) like we do in https://github.com/railsware/capybara_mock/pull/17. If yes, probably, we should simplify to strip patch versions away.